src/client.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""This file is the client module of the CF 401 Python HTTP-Server assignment."""
+
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+import socket
+
+
+def client(message=sys.argv[1]):
```

Comment:
I don't think this is a good idea....using sysargv as default parameters to a function.  You really should only be dealing with that in the if __name__ == main block...